src/server/mod.ts

```diff
@@ -60,13 +61,43 @@ export interface DenoConfig {
 
 export { ServerContext };
 
-export async function start(routes: Manifest, opts: StartOptions = {}) {
-  const ctx = await ServerContext.fromManifest(routes, opts);
+export async function start(
+  routes: Manifest,
+  opts: StartOptions | TlsStartOptions = {},
```

Comment:
I would prefer to have two separate `start(..., opts: StartOptions)` and `startTls(..., opts: StartTlsOptions)`. I would be happy to create a new PR.

Reply:
I think we should keep the one function, `start()`, that accepts both interfaces. This keeps it in line with `Deno.serve()`, which will soon be made stable.

Reply:
Ah. I missed that `Deno.serveTls()` doesn't exist.

Anyway, if we are following `Deno.serve()` pattern, and not `serve()`/`serveTls()` pattern from `std/http/server.ts`, it makes sense to keep one `start()`.

Reply:
For context, `serve()` and `serveTls()` from `std` look like they'll be removed once `Deno.serve()` is stabilised (PR [here](https://github.com/denoland/deno_std/pull/3381)).